mappings/net/minecraft/container/AbstractFurnaceContainer.mapping

```diff
@@ -13,9 +13,14 @@ CLASS asx net/minecraft/container/AbstractFurnaceContainer
 	METHOD b sendContentUpdates ()V
 	METHOD b setProperty (II)V
 		ARG 1 pos
-	METHOD b transferSlot (Larb;I)Lawo;
-		ARG 1 player
 	METHOD d clearCraftingSlots ()V
 	METHOD e getCraftingResultSlotIndex ()I
 	METHOD f getCraftingWidth ()I
 	METHOD g getCrafitngHeight ()I
+	METHOD b transferSlot (Larb;I)Lawo;
+		ARG 1 player
+	METHOD b isFuel (Lawo;)Z
+		ARG 1 itemStack
+	METHOD a canCook (Lawo;)Z
```

Comment:
canCook or canSmelt?

Reply:
canProcess IMO, as it's used for smelting, blasting, and smoking

Reply:
canProcess.